spk/shairport/src/dsm-control.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport"
+DNAME="ShairPort"
+# Others
+SHAIRPORT="${SYNOPKG_PKGDEST}/bin/shairport"
+VAR_DIR="${SYNOPKG_PKGDEST}/var"
+PID_FILE="${VAR_DIR}/${PACKAGE}.pid"
+
+
+start_daemon ()
+{
+    # Launch the service in the background.
+	cd ${SYNOPKG_PKGDEST}
+	$SHAIRPORT --daemon --pidfile="$PID_FILE" --log=$SYNOPKG_TEMP_LOGFILE
+    # Wait until the service  is ready (race condition here).
+    counter=5
+    while [ $counter -gt 0 ]
+    do
+        daemon_status && break
+        let counter=counter-1
+        sleep 1
+    done
+}
+
+stop_daemon ()
+{
+    # Kill the servive.
+    kill -9 `cat ${PID_FILE}`
+
+    # Wait until transmission is really dead (may take some time).
```

Comment:
Also `kill -9` is really rude. What about using the standard `stop_daemon` here?